### ISSUE TITLE
Jetpacks from Simply Jetpacks 2 have their directional flight speeds reversed.

### ISSUE THREAD

**Issue by finladdie** [FTBTeam/FTB-Continuum#499]
* **Modpack Version**: 1.5.2

* **Issue**: This same issue has been stated in many ways but I have a solution. 

The jetpacks from Simply Jetpacks 2 have their values for "Sideways Speed" accidentally assigned to "Vertical Speed" and vice versa. When you fly with any SimplyJetpacks jetpack, the vertical flight speed is very fast but the horizontal speed is slow. I know from experience that the jetpacks from SimplyJetpacks are supposed to have a very fast horizontal speed and a relatively slow vertical speed. 

In addition to this, the hover speeds are upside down. Enabling hover makes you ascend slow and descend very fast and sneaking makes the descending slower. This is because the value for "Vertical Speed (Hover Mode / Slow Descent)" has been assigned to the value for "Vertical Speed (Hover Mode)".

* **Link to Log or Crash File Paste**: N/A

* **Is it Repeatable?**: This issue is constantly present while using the said jetpacks.

* **Mod/s Affected**: Simply Jetpacks 2, version 2.2.7.45

* **Known Fix**: I took a look at the mod config and found that the speed values have simply been entered the wrong way around. The values can be directly changed from the mod config. NOTE: the top tier jetpacks (creative and Flux-infused jetpacks) have their HOVER values the right way around (their normal flight speeds are still reversed) so those specific values don't need fixing.

**Comment by blizzard4591** [comment]
@Finladdie Thank you for posting this! Could you explain the necessary changes (or post your configs)? I have the same problem (pack version 1.5.2), but I do not know what the "correct values should be. 

For the Tier 1 pack, the config lists

> D:"Sideways Speed"=0.0

When switching this with the Vertical Speed, would the jetpack no longer go upwards? This seems wrong.
Sorry for bothering you!

**Comment by finladdie** [comment]
@blizzard4591 That is actually quite strange. Even on the official FTB wiki both existing tier 1 jetpacks have a sideways speed of 0.0.

Anyway, I compared the configs of FTB Continuum (version 1.5.2) and FTB Infinity Evolved Skyblock (version 2.0.1), a modpack that has the same config values as stated in the FTB wiki. I also went in-game to test the jetpacks.

The higher tier jetpacks feel so slow in Continuum because their "Sprint Speed Multiplier" is set to 1.0. Normally the multiplier is increased in tiers 3, 4 and 5. Additionally, the hover speeds were all over the place and the right values are written on the FTB wiki page.

I'm very busy at the moment so I can at least give you my simplyjetpacks.cfg (as a text file) from FTB Infinity Evolved Skyblock. It is virtually useless to you but it has the right values provided you are willing to search. Sorry in advance.

[simplyjetpackscfg.txt](https://github.com/FTBTeam/FTB-Continuum/files/2733445/simplyjetpackscfg.txt)
